types/echarts/index.d.ts

```diff
@@ -1170,9 +1170,9 @@ declare namespace echarts {
                  * Moreover, it can be set as negative number, like `-3`.
                  *
                  * @default null
-                 * @see https://ecomfe.github.io/echarts-doc/public/en/option.html#yAxis.min
+                 * @see https://echarts.apache.org/zh/option.html#yAxis.min
```

Comment:
I recommend that the the link should point to the English version: e.g. `/en/` instead of `/zh/`

Reply:
You may use `https://echarts.apache.org/option.html#yAxis.min` because it will redirect to the English or Chinese version based on the language setting of the Web browser.